docs/cron/export.sh

```diff
@@ -74,9 +74,9 @@ mysql --skip-column-names --batch tatoeba -e \
 # split user languages by language
 mysql --skip-column-names --batch tatoeba -e \
     "SELECT
-       COALESCE(ul.language_code, '\N'), 
-       COALESCE(ul.level, '\N'), 
-       COALESCE(u.username, '\N'), 
+       COALESCE(ul.language_code, '\\N'), 
```

Comment:
We currently don't have rows where `language_code` is `NULL` but we have ones where it is the empty string:
```
root@sloth:~# mariadb tatoeba -e "SELECT u.username, u.since, u.last_time_active, ul.created, ul.modified from users_languages ul join users u on ul.of_user_id = u.id where ul.id in (select id from users_languages where language_code = '');"
+----------------+---------------------+------------------+---------------------+---------------------+
| username       | since               | last_time_active | created             | modified            |
+----------------+---------------------+------------------+---------------------+---------------------+
| Epoxi117       | 2015-09-01 01:13:02 |       1441073214 | 2015-09-01 02:06:55 | 2015-09-01 02:06:55 |
| battalgazi     | 2015-09-01 19:44:06 |       1441313185 | 2015-09-03 20:56:09 | 2015-09-03 20:56:09 |
| johngb         | 2015-09-24 14:58:26 |       1443111814 | 2015-09-24 16:23:34 | 2015-09-24 16:23:34 |
| saluyot        | 2015-10-21 08:55:57 |       1447690193 | 2015-10-21 09:03:07 | 2015-10-21 09:03:07 |
| Nafas          | 2015-10-24 05:51:58 |                0 | 2015-10-24 05:57:53 | 2015-10-24 05:57:53 |
| iceqween000    | 2015-10-25 01:07:57 |       1445737356 | 2015-10-25 01:42:36 | 2015-10-25 01:42:36 |
| anhsontran     | 2015-11-06 05:44:38 |       1446793285 | 2015-11-06 07:07:42 | 2015-11-06 07:07:42 |
| Wermesondavid  | 2016-01-31 22:41:51 |       1455911625 | 2016-01-31 22:51:46 | 2016-01-31 22:51:46 |
| ysnbibi        | 2016-02-28 22:10:55 |       1463436734 | 2016-02-28 23:01:43 | 2016-02-28 23:01:43 |
| Hagensbarn     | 2016-03-17 11:34:52 |       1458215862 | 2016-03-17 11:49:16 | 2016-03-17 11:49:16 |
| LibertytoDream | 2016-04-30 01:44:01 |       1461982383 | 2016-04-30 02:50:43 | 2016-04-30 02:50:43 |
| ibrahimovic    | 2016-12-13 07:15:06 |       1481812605 | 2016-12-13 07:21:12 | 2016-12-13 07:21:12 |
| ksolina        | 2017-08-12 12:35:14 |                0 | 2017-08-12 13:03:43 | 2017-08-12 13:03:43 |
| hiro           | 2017-09-03 15:50:06 |                0 | 2017-09-03 15:51:37 | 2017-09-03 15:51:37 |
| ikben1970      | 2017-09-24 21:56:14 |                0 | 2017-09-24 22:03:03 | 2017-09-24 22:03:03 |
| tjitske        | 2018-03-06 09:44:47 |                0 | 2018-03-06 09:46:29 | 2018-03-06 09:46:29 |
| Sirimiri1      | 2018-09-03 12:50:26 |                0 | 2018-09-03 17:39:44 | 2018-09-03 17:39:44 |
+----------------+---------------------+------------------+---------------------+---------------------+
```
@jiru @trang I guess we should fix this (either set the language to null or even remove the unknown language for these (inactive) users). Otherwise it will probably break the script.

But the escaping issue still stays and even after trying a few things and searching the web I haven't found a solution how to send from the shell a properly escaped backslash combination that is correctly interpreted by MariaDB.
So I think it's easier to use a special value like `NULL` and pass that to the awk script which would convert it to `\N`.

EDIT: I've finally found a way to get `\N` back from the query:
```
$ mariadb --skip-column-names --batch --raw tatoeba -e "select coalesce(null, '\\\N');"
\N
```